src/rprblender/export/image.py

```diff
@@ -122,7 +123,7 @@ def sync(rpr_context, image: bpy.types.Image, use_color_space=None, frame_number
 
 def set_image_gamma(rpr_image, image, color_space, rpr_context):
    # RPRImageTexture node color space names are in caps, unlike in Blender
-    if isinstance(rpr_context, context.RPRContext2):
+    if isinstance(rpr_context, (context.RPRContext2, RPRContextHybridPro)):
```

Comment:
Is this check necessary anymore?  I thought we removed old modes?

Reply:
This check is made for High, Medium, and Low.

Reply:
We still support Hybrid only if HybridPro isn't supported
https://github.com/GPUOpen-LibrariesAndSDKs/RadeonProRenderBlenderAddon/blob/develop/src/rprblender/engine/__init__.py#L103
Therefore such check is required.